### PR TITLE
[Integ-tests-3.10.0] Fix integration tests

### PR DESCRIPTION
**Pull request by hanwen-pcluste**
### Description of changes
* Describe *what* you're changing and *why* you're doing these changes.

### Tests
* Describe the automated and/or manual tests executed to validate the patch.
* Describe the added/modified tests.

### References
* Link to impacted open issues.
* Link to related PRs in other packages (i.e. cookbook, node).
* Link to documentation useful to understand the changes.

### Checklist
- Make sure you are pointing to **the right branch**.
- If you're creating a patch for a branch other than `develop` add the branch name as prefix in the PR title (e\.g\. `[release-3.6]`).
- Check all commits' messages are clear, describing what and why vs how.
- Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integration-tests/configs/released.yaml [hunk 1]
```diff
@@ -170,11 +170,11 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
           oss: ["ubuntu2004"]
           schedulers: ["slurm"]
-    test_api.py::test_custom_image:
-      dimensions:
-        - regions: ["sa-east-1"]
-          instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: ["alinux2"]
+#    test_api.py::test_custom_image:
```
Comment (gmarciani) [inline tests/integration-tests/configs/released.yaml]: Why are you commenting out this test?

File: tests/integration-tests/configs/released.yaml [hunk 2]
```diff
@@ -60,12 +60,12 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
           oss: ["rhel8"]  # os must be different from ubuntu2004 to test os validation logic when wrong os is provided
           schedulers: ["slurm"]
-    test_create.py::test_create_wrong_pcluster_version:
-      dimensions:
-        - regions: ["ca-central-1"]
-          instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: ["alinux2"]
-          schedulers: ["slurm"]
+#    test_create.py::test_create_wrong_pcluster_version:
```
Comment (gmarciani) [inline tests/integration-tests/configs/released.yaml]: Why are you commenting out this test?

File: tests/integration-tests/configs/released.yaml (hunk 1 above)
Comment (hanwen-pcluste) [reply tests/integration-tests/configs/released.yaml]: the commit descriptions have the details https://github.com/aws/aws-parallelcluster/pull/6327/commits

File: tests/integration-tests/configs/released.yaml (hunk 2 above)
Comment (hanwen-pcluste) [reply tests/integration-tests/configs/released.yaml]: the commit descriptions have the details https://github.com/aws/aws-parallelcluster/pull/6327/commits

**Review by gmarciani — APPROVED** [review]

**Pull request merged**